src/modes/battle/battle.cpp

```diff
@@ -540,11 +540,14 @@ void BattleMode::ChangeState(BATTLE_STATE new_state)
 
         // Display a message about the stamina bonus related event
         if (_hero_init_boost && _enemy_init_boost)
-            GetIndicatorSupervisor().AddShortNotice(UTranslate("Double Rush!"), "data/gui/menus/star.png");
```

Comment:
Here I disagree. It is a bad practice to have multiple line if result without braces. If you absolutely want to add a new line here, please add braces.